pom.xml

```diff
@@ -1021,6 +1022,7 @@
                 <module>json-modules</module>
                 <module>jsoup</module>
                 <module>jws</module>
+                <module>kafka-elastic</module>
```

Comment:
The other kafka modules are named apache-kafka-N so let's name this one apache-kafka-elastic

Reply:
incoprated